api/client/options.proto

```diff
@@ -3,44 +3,59 @@ syntax = "proto3";
 package nighthawk.client;
 
 import "google/protobuf/duration.proto";
-import "validate/validate.proto";
+import "google/protobuf/wrappers.proto";
+import "envoy/api/v2/auth/cert.proto";
 import "envoy/api/v2/core/base.proto";
+import "validate/validate.proto";
 
 message RequestOptions {
   envoy.api.v2.core.RequestMethod request_method = 1;
   repeated envoy.api.v2.core.HeaderValueOption request_headers = 2;
-  uint32 request_body_size = 3 [(validate.rules).uint32 = {lte: 4194304}];
+  google.protobuf.UInt32Value request_body_size = 3 [(validate.rules).uint32 = {lte: 4194304}];
 }
 
 // TODO(oschaaf): Ultimately this will be a load test specification. The fact that it
 // can arrive via CLI is just a concrete detail. Change this to reflect that.
 message CommandLineOptions {
   // See :option:`--rps` for details.
-  uint64 requests_per_second = 1 [(validate.rules).uint64 = {gte: 1, lte: 1000000}];
+  google.protobuf.UInt32Value requests_per_second = 1
+      [(validate.rules).uint32 = {gte: 1, lte: 1000000}];
   // See :option:`--connections` for details.
-  uint64 connections = 2 [(validate.rules).uint64 = {gte: 1, lte: 1000000}];
+  google.protobuf.UInt32Value connections = 2 [(validate.rules).uint32 = {gte: 1, lte: 1000000}];
   // See :option:`--duration` for details.
   google.protobuf.Duration duration = 3 [(validate.rules).duration.gte.nanos = 1];
   // See :option:`--timeout` for details.
   google.protobuf.Duration timeout = 4 [(validate.rules).duration.gte.seconds = 0];
   // See :option:`--h2` for details.
-  bool h2 = 5;
+  google.protobuf.BoolValue h2 = 5;
```

Comment:
Generally in Envoy, these should be scalar if the default is empty/0/"" and wrapped if you need something else. Is this the case for all of these?

Reply:
My 2p: While we may get away with scalar booleans today, in the future we may want to allow default configurations to override the ones we provide in code. When that happens, I think wrapping the booleans as well is warranted, because we can discern between "not provided" and 'false' values on incoming configurations, before we override the existing one.